utils/codegen/codegen-py/templates/convertSequenceType.h.jinja2

```diff
@@ -26,85 +26,85 @@ SOFTWARE.
 
 #pragma once
 
-#include <etsi_its_{{ etsi_type }}_coding/{{ etsi_type }}_{{ t_name }}.h>
+#include <etsi_its_{{ etsi_type }}_coding/{{ etsi_type }}_{{ asn1_type_name }}.h>
```

Comment:
use `{{x}}` instead of `{{ x }}` for all jinja templates

Reply:
Done with cc460684aa93d203e84fbc8c32403245c3bc1605.